Description: Use _wctime instead of wctime
Author: Kai Tietz <kai.tietz@onevision.com>
Origin: http://mingw-w64.svn.sourceforge.net/viewvc/mingw-w64?view=revision&revision=4300

--- mingw-w64-1.0+20110723.orig/mingw-w64-crt/ChangeLog
+++ mingw-w64-1.0+20110723/mingw-w64-crt/ChangeLog
@@ -1,3 +1,10 @@
+2011-07-26  Ozkan Sezer  <sezeroz@gmail.com>
+
+	Merge from trunk rev. 4299:
+	2011-07-26  Kai Tietz  <kai.tietz@onevision.com>
+	* misc/_wctime32.c: Fix symbol to use _wctime instead of wctime
+	(which isn't present).
+
 2011-07-23  Ozkan Sezer  <sezeroz@gmail.com>
 
 	Merged from trunk rev. 4284:
--- mingw-w64-1.0+20110723.orig/mingw-w64-crt/misc/_wctime32.c
+++ mingw-w64-1.0+20110723/mingw-w64-crt/misc/_wctime32.c
@@ -8,10 +8,10 @@
 #endif
 #include <time.h>
 
-extern wchar_t * (__cdecl *__MINGW_IMP_SYMBOL(wctime))(const __time32_t *);
+extern wchar_t * (__cdecl *__MINGW_IMP_SYMBOL(_wctime))(const __time32_t *);
 
 wchar_t *__cdecl _wctime32(const __time32_t *_Time)
 {
-  return (*__MINGW_IMP_SYMBOL(wctime))(_Time);
+  return (*__MINGW_IMP_SYMBOL(_wctime))(_Time);
 }
 
