Description: Add missing / to open boilerplate comments
Author: Stephen Kitt <skitt@debian.org>
Origin: https://bugs.debian.org/761191

diff --git a/mingw-w64-headers/include/adtgen.h b/mingw-w64-headers/include/adtgen.h
index b1878c1..d11f5c2 100644
--- a/mingw-w64-headers/include/adtgen.h
+++ b/mingw-w64-headers/include/adtgen.h
@@ -1,7 +1,7 @@
-**
-* This file is part of the mingw-w64 runtime package.
-* No warranty is given; refer to the file DISCLAIMER within this package.
-*/
+/**
+ * This file is part of the mingw-w64 runtime package.
+ * No warranty is given; refer to the file DISCLAIMER within this package.
+ */
 
 #include <winapifamily.h>
 
diff --git a/mingw-w64-headers/include/authz.h b/mingw-w64-headers/include/authz.h
index 64c3057..d6dd14b 100644
--- a/mingw-w64-headers/include/authz.h
+++ b/mingw-w64-headers/include/authz.h
@@ -1,7 +1,7 @@
-**
-* This file is part of the mingw-w64 runtime package.
-* No warranty is given; refer to the file DISCLAIMER within this package.
-*/
+/**
+ * This file is part of the mingw-w64 runtime package.
+ * No warranty is given; refer to the file DISCLAIMER within this package.
+ */
 
 #include <winapifamily.h>
 
diff --git a/mingw-w64-headers/include/aux_ulib.h b/mingw-w64-headers/include/aux_ulib.h
index 71a74c2..3fac7ae 100755
--- a/mingw-w64-headers/include/aux_ulib.h
+++ b/mingw-w64-headers/include/aux_ulib.h
@@ -1,7 +1,7 @@
-**
-* This file is part of the mingw-w64 runtime package.
-* No warranty is given; refer to the file DISCLAIMER within this package.
-*/
+/**
+ * This file is part of the mingw-w64 runtime package.
+ * No warranty is given; refer to the file DISCLAIMER within this package.
+ */
 
 #ifndef _AUX_SHLD_LIB_H
 #define _AUX_SHLD_LIB_H
diff --git a/mingw-w64-headers/include/avifmt.h b/mingw-w64-headers/include/avifmt.h
index 88389a3..3b8f2c7 100644
--- a/mingw-w64-headers/include/avifmt.h
+++ b/mingw-w64-headers/include/avifmt.h
@@ -1,7 +1,7 @@
-**
-* This file is part of the mingw-w64 runtime package.
-* No warranty is given; refer to the file DISCLAIMER within this package.
-*/
+/**
+ * This file is part of the mingw-w64 runtime package.
+ * No warranty is given; refer to the file DISCLAIMER within this package.
+ */
 
 #include <winapifamily.h>
 
diff --git a/mingw-w64-headers/include/aviriff.h b/mingw-w64-headers/include/aviriff.h
index cb334ad..30e7de3 100755
--- a/mingw-w64-headers/include/aviriff.h
+++ b/mingw-w64-headers/include/aviriff.h
@@ -1,7 +1,7 @@
-**
-* This file is part of the mingw-w64 runtime package.
-* No warranty is given; refer to the file DISCLAIMER within this package.
-*/
+/**
+ * This file is part of the mingw-w64 runtime package.
+ * No warranty is given; refer to the file DISCLAIMER within this package.
+ */
 
 #include <winapifamily.h>
 
