Description: Correct extern "C" declarations
Origin: http://mingw-w64.svn.sourceforge.net/viewvc/mingw-w64?view=revision&revision=4548

--- mingw-w64.orig/mingw-w64-headers/include/imagehlp.h
+++ mingw-w64/mingw-w64-headers/include/imagehlp.h
@@ -16,10 +16,6 @@
 
 #include <wintrust.h>
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
 #include <psdk_inc/_dbg_LOAD_IMAGE.h>
 
   typedef enum _IMAGEHLP_STATUS_REASON {
@@ -45,6 +41,10 @@
     BindForwarderNOT64
   } IMAGEHLP_STATUS_REASON;
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
   typedef WINBOOL (WINAPI *PIMAGEHLP_STATUS_ROUTINE)(IMAGEHLP_STATUS_REASON Reason,PCSTR ImageName,PCSTR DllName,ULONG_PTR Va,ULONG_PTR Parameter);
   typedef WINBOOL (WINAPI *PIMAGEHLP_STATUS_ROUTINE32)(IMAGEHLP_STATUS_REASON Reason,PCSTR ImageName,PCSTR DllName,ULONG Va,ULONG_PTR Parameter);
   typedef WINBOOL (WINAPI *PIMAGEHLP_STATUS_ROUTINE64)(IMAGEHLP_STATUS_REASON Reason,PCSTR ImageName,PCSTR DllName,ULONG64 Va,ULONG_PTR Parameter);
@@ -103,9 +103,10 @@
   WINBOOL IMAGEAPI UpdateDebugInfoFile(PCSTR ImageFileName,PCSTR SymbolPath,PSTR DebugFilePath,PIMAGE_NT_HEADERS32 NtHeaders);
   WINBOOL IMAGEAPI UpdateDebugInfoFileEx(PCSTR ImageFileName,PCSTR SymbolPath,PSTR DebugFilePath,PIMAGE_NT_HEADERS32 NtHeaders,DWORD OldChecksum);
 
-#include <psdk_inc/_dbg_common.h>
-
 #ifdef __cplusplus
 }
 #endif
+
+#include <psdk_inc/_dbg_common.h>
+
 #endif
