Description: Handle libgomp initialisation
Author: Mattias Engdegård <mattiase@acm.org>
Origin: https://sourceforge.net/p/mingw-w64/mailman/message/34353737/

--- a/mingw-w64-libraries/winpthreads/src/mutex.c
+++ b/mingw-w64-libraries/winpthreads/src/mutex.c
@@ -56,7 +56,10 @@
 static bool
 is_static_initializer(pthread_mutex_t m)
 { 
-  return (uintptr_t)m >= (uintptr_t)-3;
+  /* Treat 0 as a static initializer as well (for normal mutexes),
+     to tolerate sloppy code in libgomp. (We should rather fix that code!) */
+  intptr_t v = (intptr_t)m;
+  return v >= -3 && v <= 0;
 }
 
 /* Create and return the implementation part of a mutex from a static
