Description: trivial spelling fix
Author: Stephen Kitt <steve@sk2.org>

--- mingw-w64.orig/mingw-w64-tools/gendef/src/gendef.c
+++ mingw-w64/mingw-w64-tools/gendef/src/gendef.c
@@ -114,7 +114,7 @@
     {
       if (!next)
         {
-	  fprintf (stderr, "Error: %s expects path as next arguement.", opts);
+	  fprintf (stderr, "Error: %s expects path as next argument.", opts);
 	  return 0;
         }
       gendef_addpath_def (next);
--- mingw-w64.orig/mingw-w64-libraries/libmangle/src/m_ms.c
+++ mingw-w64/mingw-w64-libraries/libmangle/src/m_ms.c
@@ -685,7 +685,7 @@
               return m_oper (c, m_rtti (c, "complete_object_locator"));
           }
         DEC_CHAR (c);
-        fprintf (stderr, " *** Unkown RTTI %c\n", ch);
+        fprintf (stderr, " *** Unknown RTTI %c\n", ch);
         c->err = 2;
         return NULL;
       case 'S':
--- mingw-w64.orig/mingw-w64-tools/genpeimg/src/img_pe.c
+++ mingw-w64/mingw-w64-tools/genpeimg/src/img_pe.c
@@ -282,7 +282,7 @@
     case 14: fprintf (outfp, "XBOX\n"); break;
     case 16: fprintf (outfp, "Windows Boot Application (16)\n"); break;
     default:
-      fprintf (outfp, "Unkown (%u)\n", PEIMG_GET_USHORT (ppeimg, ppeimg->optional_hdr_pos + 68));
+      fprintf (outfp, "Unknown (%u)\n", PEIMG_GET_USHORT (ppeimg, ppeimg->optional_hdr_pos + 68));
       break;
     }
   hdr_cha = PEIMG_GET_USHORT (ppeimg, ppeimg->optional_hdr_pos + 70);
