Description: Replace the non-reentrant strtok_r with strtok_s
Author: Stephen Kitt <skitt@debian.org>
Forwarded: https://sourceforge.net/p/mingw-w64/bugs/364/

--- a/mingw-w64-headers/crt/sec_api/string_s.h
+++ b/mingw-w64-headers/crt/sec_api/string_s.h
@@ -56,6 +56,8 @@
   _CRTIMP errno_t __cdecl _wcsset_s_l(wchar_t *_Str,size_t _SizeInChars,unsigned int _Val,_locale_t _Locale);
   _CRTIMP errno_t __cdecl _wcsnset_s_l(wchar_t *_Str,size_t _SizeInChars,unsigned int _Val, size_t _Count,_locale_t _Locale);
 
+#define strtok_r strtok_s
+
 #endif
 
 #ifndef _MEMORY_S_DEFINED
--- a/mingw-w64-libraries/winpthreads/include/pthread.h
+++ b/mingw-w64-libraries/winpthreads/include/pthread.h
@@ -452,9 +452,6 @@
 						   strcpy((_Buf),___tmp_tm);\
 						___tmp_tm;	})
 
-#ifndef strtok_r
-#define strtok_r(__s, __sep, __last)  (*(__last) = strtok((__s), (__sep)))
-#endif
 #ifndef rand_r
 #define rand_r(__seed) (__seed == __seed ? rand () : rand ())
 #endif
